linode/manager
# chore: Clean up `src/utilities` exports

## Description 📝
- Removes default exports in favor of named exports ✏️
- Removes some unused helper functions 🗑️
- No logic changes or refactors, only find and replace of imports and exports 🚫

## How to test 🧪

### Verification steps 
- Verify all Github Actions checks pass ✅
- Verify e2es pass 🤞🏼
- Open Cloud Manager and perform an overall check of the application for defects 👁️

## As an Author I have considered 🤔

*Check all that apply*
